docs/pfSense-install-guide.md

```diff
@@ -17,41 +17,18 @@ pfSense Installation Guide
 ## pfSense Image
 
 In order to install pfSense on apu2/3/5 platforms from USB, obtain following
-[Image](https://sgpfiles.pfsense.org/mirror/downloads/pfSense-CE-memstick-serial-2.4.2-RELEASE-amd64.img.gz)
-from official mirror.
-
-Extract the image somewhere and burn it onto usb stick using `dd`:
-
-```
-dd if=pfSense-CE-memstick-serial-2.4.2-RELEASE-amd64.img of=/dev/sdx status=progress
-```
-
-where `sdx` is the correct USB drive.
-
-> On BIOS v4.6.4 or newer and v4.0.14 or newer one will have to deal with double
-> console redirection in newer BIOSes due to multiconsole option in installer.
-> In the [Installation section](#installation) there is a description how to fix
-> it. If one has a write access to the stick, just edit `/boot.config` to
-> `-S115200 -h` and in `/boot/loader.conf` change
```

Comment:
@NanoCaiordo yes, it seems to be set correctly now. But I got now:
```
Starting file system checks:
eval: fsck: Permission denied
Unknown error 126; help!
ERROR: ABORTING BOOT (sending SIGTERM to parent)!
Feb  8 14:57:03 init: /bin/sh on /etc/rc terminated abnormally, going to single user mode
Enter full pathname of shell or RETURN for /bin/sh:
# fsck
-sh: fsck: Permission denied
# su
root@:/ # fsck
/sbin/fsck: Permission denied.
```
when the installer kernel boots. Have You experienced something similar?
Even entering the shell and running the fsck as root gives permission denided.

Reply:
I never encountered this particular issue but it could be caused by the install media. How did you burn the image, and under which APU firmware are you.

Reply:
I have checked v4.9.0.1 apu2c4. I had mSATA Hoodisk 16GB and a HDD Samsung ST500LM012 connected to SATA. The second one probably caused problems.